src/desktop/lan.desktop

```diff
@@ -6,12 +6,12 @@ X-KDE-ModuleType=Library
 X-KDE-HasReadOnlyMode=true
 X-SuSE-YaST-Call=lan
 
-X-SuSE-YaST-Group=Network
+X-SuSE-YaST-Group=System
 X-SuSE-YaST-Argument=
 X-SuSE-YaST-RootOnly=true
 X-SuSE-YaST-AutoInst=all
 X-SuSE-YaST-Geometry=
-X-SuSE-YaST-SortKey=
+X-SuSE-YaST-SortKey=!first
```

Comment:
have it first was required in fate#318804. I personally don't care at all. However, if ```X-SuSE-YaST-SortKey``` works only in ncurses then it make sense to have it sorted alphabetically to have same appearance in both ncurses even qt.